voucherify-service/src/commercetools/store-actions/cart-update-actions/addDirectDiscountWithDiscountSummary.ts

```diff
@@ -5,6 +5,13 @@ import {
 } from './CartAction';
 import { FREE_SHIPPING_UNIT_TYPE } from '../../../consts/voucherify';
 
+const sortDiscounts = (discounts) =>
+  discounts.sort((discount1, discount2) => {
+    if (discount1?.target?.predicate === 'true') return 1;
+    if (discount2?.target?.predicate === 'true') return -1;
```

Comment:
Is a second condition `if (discount2?.target?.predicate === 'true') return -1;` redundant as you return -1 anyway at the end of the function?